### PR TITLE
INDY-1358: Payment txns in performance test

### PR DESCRIPTION
**Pull request by spivachuk**
- Added support for payment, verify_payment and get_payment_sources transactions to the performance test script.

### CODE REVIEW

File: scripts/performance/perf_processes.py [hunk 1]
```diff
@@ -265,6 +304,9 @@ def _gen_req_data(self):
     async def on_batch_completed(self, pool_handle, wallet_handle, submitter_did):
         pass
 
+    async def on_request_replied(self, req, resp_or_exp):
```
[the remark this answers is not in the document]
Comment (spivachuk) [reply scripts/performance/perf_processes.py]: Done.

**Comment by spivachuk** [comment]
test this please

File: scripts/performance/perf_processes.py [hunk 2]
```diff
@@ -608,14 +661,260 @@ class RGGetRevocRegDelta(RGGetEntryRevoc):
...
+
+    async def _gen_req(self, submit_did, req_data):
+        payment_address = req_data
+        req, _ = await payment.build_get_payment_sources_request(self._wallet_handle,
+                                                                 self._submitter_did,
+                                                                 payment_address)
+        return req
+
+
+class RGPayment(RGBasePayment):
```
Comment (ashcherbakov) [inline scripts/performance/perf_processes.py]: Should we add this to `create_req_generator`?

File: scripts/performance/perf_processes.py [hunk 3]
```diff
@@ -608,14 +661,260 @@ class RGGetRevocRegDelta(RGGetEntryRevoc):
...
+        source_infos_json = \
+            await payment.parse_get_payment_sources_response(self._payment_method,
+                                                             get_payment_sources_resp)
+        source_infos = json.loads(source_infos_json)
+        payment_sources = []
+        for source_info in source_infos:
+            payment_sources.append((source_info["source"], source_info["amount"]))
+        return payment_sources
+
+
```
Comment (ashcherbakov) [inline scripts/performance/perf_processes.py]: Please extend docs with Payment requests (write a how-to send payment txns).

File: scripts/performance/perf_processes.py (hunk 2 above)
Comment (spivachuk) [reply scripts/performance/perf_processes.py]: Added missed entries to the map.

**Pull request merged**